.github/workflows/Test.yml

```diff
@@ -83,10 +83,23 @@ jobs:
                     tools: composer:v2
                     coverage: none
 
+            -   name: Get composer cache directory
```

Comment:
Can you put it in the same order as in Deploy.yml (between checkout ans php setup)